ideascube/tests/test_views.py

```diff
@@ -363,7 +363,9 @@ def test_staff_can_set_password(staffapp, client, user):
     response = form.submit(status=302)
     assert response.location.endswith(
         reverse('user_detail', kwargs={'pk': user.pk}))
-    assert user_model.objects.get(pk=user.pk).password != old_password
```

Comment:
Hum, we use `user.password` to get the `old_password`, so the comparison is made on both encrypted password so it should be ok. Or I miss something ?